tools/main.py

```
@@ -1,9 +1,9 @@
 import openai
 import json
 import os
-from test_navigation_tool import windows_navigate, shortcuts, windows_navigate_function
-from test_edit import edit_file, edit_file_function
-from test_bash import bash_command,bash_command_function
+from windows_navigation import WindowsNavigationTool
```

Ruff (0.8.2)

`windows_navigation.WindowsNavigationTool` imported but unused

Remove unused import: `windows_navigation.WindowsNavigationTool`

(F401)